specification/servicelinker/ServiceLinker.Management/main.tsp

```diff
@@ -0,0 +1,47 @@
+/**
+ * PLEASE DO NOT REMOVE - USED FOR CONVERTER METRICS
+ * Generated by package: @autorest/openapi-to-typespec
+ * Parameters used:
+ *   isFullCompatible: true
+ *   guessResourceKey: false
+ * Version: 0.10.14
+ * Date: 2025-04-29T07:23:07.547Z
+ */
+import "@typespec/rest";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+import "./models.tsp";
+import "./DryrunResource.tsp";
+import "./LinkerResource.tsp";
+import "./routes.tsp";
+import "./back-compatible.tsp";
```

Comment:
It would be great to have consistent casing for all of these file names

Reply:
Also, the models.tsp is imported through some of these other files

Reply:
What other files you are saying? models.tsp is expected to be imported from main.tsp.